core/deployment/src/main/java/io/quarkus/deployment/configuration/type/CollectionOf.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.deployment.configuration.type;
+
+import java.util.Objects;
+
+/**
+ *
+ */
+public final class CollectionOf extends ConverterType {
```

Comment:
It's a general comment but I'm pretty sure some methods of this class are missing `@Override` annotations.

While not an issue per se, it really helps in the long run because you have an error if an extended method gets renamed/changed in the parent class. Otherwise, things just break silently.

Reply:
BTW, it's not only rhetorical, I remember tracking a bug in a large code base for a while because of this.

Reply:
Fixed locally.